### PR TITLE
feat(ui): tracking tab

### LINKED ISSUE #724: profile tracking tab
On a user's profile, show a list of all tracked projects with replicated ones at the top and tracked-but-not-yet-replicated below.

👆 [Figma prototype is here](https://www.figma.com/proto/dnYlebm0ehgyJyT3Po9Kcl/Upstream?node-id=17108%3A6076&viewport=369%2C1744%2C0.30791011452674866&scaling=min-zoom&hide-ui=1).

🖋 [Figma design file is here](https://www.figma.com/file/dnYlebm0ehgyJyT3Po9Kcl/Upstream?node-id=17042%3A259062).

image

### PR DESCRIPTION
**Pull request by sarahscott**
Screen Shot 2020-08-13 at 9 34 29 PM

Closes #724

### CODE REVIEW

File: ui/DesignSystem/Component/ProjectListItem.svelte [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<script>
```
Comment (xla) [inline ui/DesignSystem/Component/ProjectListItem.svelte:1]:
```suggestion
<script lang="ts">
```
All new components should be typescripted.

File: ui/Screen/Org/Projects.svelte [hunk 2]
```diff
@@ -26,20 +23,13 @@
...
-        showRegisteredBadge: true,
-      };
-    } else {
-      return {
-        title: orgProject.name,
-        showRegisteredBadge: true,
-      };
-    }
-  };
+  const formatMetadata = orgProject =>
```
Comment (xla) [inline ui/Screen/Org/Projects.svelte:26]: Should live in library code.

File: ui/Screen/Profile/Tracking.svelte [hunk 3]
```diff
@@ -0,0 +1,76 @@
+<script>
```
Comment (xla) [inline ui/Screen/Profile/Tracking.svelte:1]:
```suggestion
<script lang="ts">
```

**Comment by sarahscott** [comment]
@brandonhaslegs #738 mitigates some of the feedback re: untracked project page and the `TrackToggle` button, so I'll ping you once it's merged and I can incorporate it here.

**Comment by sarahscott** [comment]
@brandonhaslegs this has all functionality except that which will rely on the proxy api (e.g. removing untracked projects, navigating to them, etc).

**Comment by brandonhaslegs** [comment]
Any way to test the notification when a project has been found?

File: ui/Screen/Profile/Projects.svelte [hunk 4]
```diff
@@ -1,29 +1,48 @@
...
     const project = event.detail;
     push(path.projectSource(project.id));
   };
 
-  const projectCardProps = project => ({
-    title: project.metadata.name,
-    description: project.metadata.description,
-    showRegisteredBadge: project.registration,
-  });
+  console.log(session);
```
Comment (rudolfs) [inline ui/Screen/Profile/Projects.svelte:24]:
```suggestion
```

File: ui/src/animation.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export const fadeDuration = 200;
```
Comment (rudolfs) [inline ui/src/animation.ts:1]: We already have a `config.ts` maybe we can put this there?

**Review by rudolfs — APPROVED** [review]
Looks great! 🚀

**Review by xla — CHANGES_REQUESTED** [review]
Still looking rather good, I'm not quite sure what the meaning of the `Hoverable` component is, could you outline how it's meant to be used across the app?

File: ui/DesignSystem/Component/TrackToggle.svelte [hunk 6]
```diff
@@ -1,19 +1,17 @@
...
   export let tracking = false;
+  export let expanded = false;
 
-  let hover = false;
-  let active = false;
-
-  const enter = () => {
-    hover = true;
-  };
+  // warning variant implies consequences of untracking are more severe
```
[the remark this answers is not in the document]
Comment (sarahscott) [reply ui/DesignSystem/Component/TrackToggle.svelte:11]: It's not code-related; it's to provide context for what the variant looks like visually and why it exists. Probably makes more sense to just put this in the design guide.

**Comment by sarahscott** [comment]
@xla `Hoverable` is `Copyable` for hovering.

File: ui/src/org.test.ts [hunk 7]
```diff
@@ -1,4 +1,4 @@
-import { getOrg } from "./org";
+import * as org from "./org";
 import { idValidationStore } from "./id";
 import { ValidationStatus } from "./validation";
 import { get } from "svelte/store";
```
Comment (xla) [inline ui/src/org.test.ts:4]:
```suggestion
import { get } from "svelte/store";

import * as org from "./org";
import { idValidationStore } from "./id";
import { ValidationStatus } from "./validation";
```

**Review by xla — CHANGES_REQUESTED** [review]
Looks like `animation.ts` is orphaned now and can go, also not sure we should add any more to the existing org tests, given we removed the orgs for now.

**Comment by sarahscott** [comment]
Oh woops 😆 yeah `animation.ts` can definitely go. Should we keep the org unit tests at all? I'd say they provide some nice jest + svelte examples but are otherwise irrelevant if we're not bringing orgs back.

**Comment by xla** [comment]
> Oh woops laughing yeah `animation.ts` can definitely go. Should we keep the org unit tests at all? I'd say they provide some nice jest + svelte examples but are otherwise irrelevant if we're not bringing orgs back.

Good point, let's leave them until we have more certainty what the future holds for the registry.

**Review by xla — APPROVED** [review]
🗂 4️⃣ ☀️ 👂

**Review by rudolfs — APPROVED** [review]

**Pull request merged**